code/modules/antagonists/blob/blob.dm

```diff
@@ -47,16 +59,46 @@
 	icon_icon = 'icons/mob/blob.dmi'
 	button_icon_state = "blob"
 
-/datum/action/innate/blobpop/Activate()
+	var/autoplace_time = OVERMIND_STARTING_AUTO_PLACE_TIME
+
+/datum/action/innate/blobpop/Grant(Target)
+	. = ..()
+	if(owner)
+		addtimer(CALLBACK(src, /datum/action/innate.proc/Activate, TRUE), autoplace_time, TIMER_UNIQUE|TIMER_OVERRIDE)
```

Comment:
Can't you just do `.proc/Activate`?

Reply:
it's not a proc defined on the datum so I don't think I can.

Reply:
nevermind I can